Dictionaries/hrv_OCRFixReplaceList.xml

```diff
@@ -2030,17 +2030,18 @@
     <RegEx find="be[čć]aje" replaceWith="bećava" />
     <RegEx find="iliše" replaceWith="ilizira" />
     <RegEx find="ilišu" replaceWith="iliziraju" />
-    <RegEx find="citov" replaceWith="citir" />
     <RegEx find="ćič" replaceWith="čić" />
     <RegEx find="dilova" replaceWith="dila" />
-    <RegEx find="ditov" replaceWith="ditir" />
+    <RegEx find=([cd])ditov" replaceWith="$1itir" />
```

Comment:
Perhaps it's missing a " (quote) ?

Reply:
not only that , but I also typed wrong letter. thanks.